src/uw_basic.jl

```diff
@@ -124,14 +124,146 @@ Compute resonance frequency of a freely oscillating has bubble in water, given:
 - bubble `radius` in meters
 - `depth` of bubble in water in meters
 - gas ratio of specific heats 'γ', default: 1.4 (for air)
-- atmospheric pressure 'p0', default: 1.013e5
+- atmospheric pressure 'p0' in Pa, default: 1.013e5
 - density of water 'ρ' in kg/m³, default: 1022.476
 
 This ignores surface-tension, thermal, viscous and acoustic damping effects, and the pressure-volume relationship is taken to be adiabatic.
 Implementation based on Medwin & Clay (1998).
 """
-function bubbleresonance(radius, depth=0.0, γ=1.4, p0=1.013e5, ρ=1022.476)
+function bubbleresonance(radius::Real, depth::Real=0.0, γ::Real=1.4, p0::Real=1.013e5, ρ::Real=1022.476)
   g = 9.80665 #acceleration due to gravity
   pₐ = p0 + ρ*g*depth
   1 / (2π * radius) * √(3γ * pₐ/ρ)
 end
+
+"""
+$(SIGNATURES)
+Compute bubble resonant frequency and damping constants. Uses Prosperetti (1977) equations to compute damping coefficients and natural frequency.
+(Thermal effects and damping mechanisms in the forced radial oscillations of gas bubbles in liquids).
+
+Parameters:
+- bubble 'radius' in meters
+- frequency 'f' in Hz
+- water depth 'z' in meters. default: 0
+- bubble gas and water temperature 'T' in Kelvin. default: 293.15
+- atmospheric pressure 'p0' in Pa. default: 101.3e5
+- 'gas' number. If 2: Oxygen, 3: Nitrogen, 4: Methane, any other number: Air. default: 1
+- Soundspeed 'c' in m/s. default: 1500
+- seawater density 'ρ' in kg m^-3. default: 1022.476
+
+Returns a 1x4 Array{Float64}
```

Comment:
Why all the strong typing in this function? It'll make it hard for auto-diff etc. Please avoid typing unless necessary for dispatch. If they are required for performance, try and make them inferred/computed types and not hardcode them as `Float64`